### ISSUE TITLE
Diagram on web

### ISSUE THREAD

**Issue by leandrodamascena** [mingrammer/diagrams#177]
Ok, I know that there is already an issue for this, but I opened a new one as there is already a case study.

First of all I would like to say that I am not experienced in CSS / HTML (I don't like it) and we will need support from those who are experienced if we are to develop this.

I made a prototype of how we can develop the Diagram model on the web ... The case study involves:
1 - HTML page
2 - Editor
3 - Flask backend to process diagram

The user can write the diagram code as he writes it in a .py file and click to generate the diagram. JQuery makes a request with the code (we must implement a security layer later) and Flask executes this code returning a base64 from the diagram created.

We can implement this using Docker and a dynamic container for each request or even use Flask.
We can implement site authentication, drag and drop, premium features, many things.

Some changes will necessary in Diagrams core to reach this goal..

Watch the video and Let me know what you think..

https://streamable.com/m0atyt

**Comment by gabriel-tessier** [comment]
@leandrodamascena 

Thanks for the comment and interest on my PR. 
I wonder as I put in the other issue if it's not better to have a different project but I prefer to host in an organization with diagrams than having a separated fork project as I'm not sure to be able to maintain it alone. Also Diagrams already have a lot of contributors and at least 3 peoples (me included) thought about building a web tools. So it will be better to keep the projects hosted under the same organization.
Wait for @mingrammer for a comment! m(__)m

Also if you have some comments on the PR they are welcome!

I watched your video and I'll check about adding an editor for the code and fix some bugs... now it's a simple textarea! :_D

**Comment by leandrodamascena** [comment]
Yes @gabriel-tessier , I agree with you to keep the project under the same Organization.. I don’t want to fork this project and go in different way. What I mean was to contribute here and may be we should be contributors for web layer.

**Comment by leandrodamascena** [comment]
@gabriel-tessier 

And about editor it´s a simple editor from https://ace.c9.io/..

If you want, we can discuss about it in Discord and update PR with new features.

**Comment by gabriel-tessier** [comment]
Thanks for the editor info I'll check more deeply about.
About discord: gab#3734

**Comment by mingrammer** [comment]
Working on https://github.com/gabriel-tessier/diagrams-web

**Comment by bdsoha** [comment]
This seems to be a duplicate of #24.
